src/api_server/src/request/mmds.rs

```diff
@@ -5,7 +5,7 @@ use crate::parsed_request::{Error, ParsedRequest, RequestAction};
 use crate::request::Body;
 use logger::{IncMetric, METRICS};
 use micro_http::StatusCode;
-use vmm::rpc_interface::VmmAction::SetMmdsConfiguration;
+use vmm::rpc_interface::VmmAction::SetMmdsNetConfiguration;
```

Comment:
not important but shouldn't this line be in the previous commit?

Reply:
same goes for the other changes in this commit that rename vars from smth_mmds_config to smth_mmds_net_config

Reply:
Not applicable anymore.